Ska/ParseCM.py

```diff
@@ -172,4 +173,28 @@ def read_mm(filename):
     return manvr_list
 
 
+def read_si_align(char_file):
+    """
+    Read ODB_SI_ALIGN matrices from supplied characteristics file
 
+    :param char_file: OFLS ODB characteristics file
+    :returns: dictionary of matrices, one per detector in original Fortran order
```

Comment:
OK.  Should I call them something different then or are we just OK?  I was just a little concerned if the convention in the top of calc_si_align was to transpose the thing called "ODB_SI_ALIGN".